Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM node:22
```

Comment:
node:latest pls

Reply:
I would tend to favor the fixed version to promote predictability, backwards compatibility and security. Using the latest tag would promote builds to non-lts versions which could introduce unwanted/unknown issues.

When updating versions, then the Dockerfile should simply be updated as well.